tgui/packages/tgui/constants.js

```diff
@@ -185,7 +185,7 @@ export const RADIO_CHANNELS = [
     color: '#1ecc43',
   },
   {
-    name: 'Common',
+    name: ':h Common',
```

Comment:
This also feels funny.

Reply:
This is how I'd prefer to do it, since logically the radio channel names should be pulled from this list, but they're pulled from the defines instead.